src/themes/OLH/templates/journal/article.html

```diff
@@ -351,19 +351,24 @@ <h4>{% trans "Authors" %}</h4>
                             <div class="section">
                                 {% if article.is_published or proofing %}
                                     <h3>{% trans "Download" %}</h3>
-                                    <ul>
-                                        {% for galley in galleys %}
-                                            <li>
-                                                <a href="{% url 'article_download_galley' article.id galley.id %}">{% trans "Download" %} {{ galley.label }}</a>
-                                                {% if galley.file.mime_type == 'application/pdf' and journal.view_pdf_button %}
-                                                    </li>
-                                                    <li>
-                                                    <a target="_blank" href="{% url 'article_view_galley' article.id galley.id %}">{% trans 'View PDF' %}</a>
-                                                    </li>
-                                                {% endif %}
-                                            </li>
-                                        {% endfor %}
-                                    </ul>
+                                    {% if galleys %}
+                                        <ul>
+                                            {% for galley in galleys %}
+                                                <li>
+                                                    <a href="{% url 'article_download_galley' article.id galley.id %}">{% trans "Download" %} {{ galley.label }}</a>
+                                                    {% if galley.file.mime_type == 'application/pdf' and journal.view_pdf_button %}
+                                                        </li>
+                                                        <li>
+                                                        <a target="_blank" href="{% url 'article_view_galley' article.id galley.id %}">{% trans 'View PDF' %}</a>
+                                                        </li>
+                                                    {% endif %}
+                                                </li>
+                                            {% endfor %}
+                                        </ul>
+                                    {% else %}
+                                        <p> {% trans 'Downloads are not available for this article.' %}</p>
+                                    {% endif %}
+
                                     {% if proofing %}
                                         <p id="note_to_proofreader_2">
                                             <i class="fa fa-info"></i>
```

Comment:
This note to the proofreader should show up only if there are galleys, as otherwise it may be taken to mean that there may be galleys that are not showing, with the new "Downloads are not available" message. Could we put it inside `if galleys` above?

Reply:
done